vignettes/tweak-page-hooks.Rmd

```diff
@@ -0,0 +1,105 @@
+---
+title: "Custom pkgdown tweak page hooks"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Custom pkgdown tweak page hooks}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+# Introduction
+
+This article will go over the following:
+
+* The process for using custom hooks in R packages
+* Example how to create custom hooks
+
+For more information about hooks, please read `help("getHook", "base")`.
```

Comment:
Is it possible to put in the issues this is addressing for admiral?  My understanding is this is helping to solve the absolute link issue.  But it would be nice to either include the issues this solves for future people reading this vignette and them trying to understand why this was made????